hip-0000-zero-token-operations.md

```diff
@@ -0,0 +1,70 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Zero unit token operations in smart contracts
+author: littletarzan, vae.vecturne
+working-group:
+type: Standards track
+category: Service
+needs-council-approval: No
+status: Draft
+created: Sept 1, 2022
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/563
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+Integer rounding will naturally lead to DeFi contracts sometimes minting or transferring 0 units of a token. But the network rejects 0 units in these operations, forcing contract developers to include special cases that would not be necessary on most networks.
+
+## Motivation
+
+To reduce code, if statement checks, etc inside smart contracts and make DeFi contracts easier to use. To reduce bugs in DeFi applications involving math and rounding.
+
+## Rationale
+
+DeFi contracts using ERC20 standard may mint, burn, and transfer zero tokens.
+
+The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during the discussion.
```

Comment:
There is some default text here from the template hip that we should remove. Also, the rationale is your chance to really explain the reasoning behind why the hip is useful. From hip-1:

_The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages._

_The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during the discussion._